CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+- Add `Certificate::from_pem_bundle(pem_bundle: &[u8]) -> Result<Vec<Certificate>>`
```

Comment:
I don't keep unreleased changes in the change log, I put it together at release time.